src/components/GameCard.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import GameCardJacquette from './GameCardJacquette';
+import GameCardName from './GameCardName';
+import GameCardStyle from './GameCardStyle';
+import AddGameButton from './AddGameButton';
+import Plus from './Plus';
+import PlusImg from '../img/white/plus.png';
+import styled from "styled-components";
```

Comment:
do you need styled ?

Reply:
I'm refactoring all the unusued variables. PR for tonight ;)

Reply:
Pr ok